docs/guides/app-distribution.md

```diff
@@ -172,9 +172,60 @@ App app = new App(appConfig);
 **InstallationService** absorbs the difference in the response structure. So, you don't need to change anything even when you switch from the classic OAuth to the V2.
 
 
+#### Build Slack OAuth using Spring Boot
```

Comment:
Using Spring Boot is very common in the Java world. Having Spring examples here is beneficial for many.